crowdsec-docs/docs/appsec/intro.md

```diff
@@ -0,0 +1,46 @@
+---
+id: intro
+title: Introduction
+sidebar_position: 1
+---
+
+# Introduction
+
+<!-- xx : fix crowdsec version -->
+The application security component available in crowdsec:
+
+ - Provides low-effort virtual patching capabilities
+ - Supports your legacy ModSecurity rules
+ - Combines the advantages of classical WAF with the enhanced crowdsec capabilities to allow the detection of behaviors that would otherwise be difficult
+ - Is fully integrated with the Crowdsec software stack, such as the console and remediation components.
+
+<!-- xx :  links -->
+The application security component takes advantage of the remediation components existing in web servers (such as Nginx, Traefik, Haproxy etc.) to provide web application firewall capabilities:
+
+![appsec-global](/img/appsec-global.png)
+
+## Request inspection
+
+1. The remediation component - if the application security component is enabled - forwards the incoming request to the Crowdsec Security engine.
+2. The security engine will first evaluate the inband rules, meant to detect requests that must be blocked. As soon as these rules are processed, an answer is provided to the remediation component:
+   1. If no inband rule is triggered, the processing of the request will continue on the webserver side as usual
+   2. If an inband rule is triggered, the remediation component will answer with a 403 or a captcha request to the user of the incriminated request, stopping the request processing.
+3. In the background, the security engine will then evaluate the out-of-band rules. out-of-band rules do not impact performance nor response time, as they are evaluated after the aplication security component instructs the webserver to stop or go on with the request processing.
```

Comment:
### Security Engine: out-of-band rules processing
In the background, the security engine will then evaluate the out-of-band rules. These rules do not impact performance nor response time, as they are evaluated after the **AppSec component** instructs the webserver to continue or stop processing the request.